lib/clickhousex/codec/json.ex

```diff
@@ -99,6 +102,14 @@ defmodule Clickhousex.Codec.JSON do
     String.to_integer(value)
   end
 
+  defp to_native("Decimal" <> _, value) when @use_decimal and is_bitstring(value) do
+    Decimal.new(value)
+  end
+
+  defp to_native("Decimal" <> _, value) when is_bitstring(value) do
+    String.to_float(value)
```

Comment:
Why do you have a different types `float / decimal`?
I think we should use only one type.

We also get float or integer types in json, like example where I sent you

Reply:
Because floats and decimals are 2 different types in clickhouse. And I don't want to enforce usage of decimals for this lib, so floats are always converted to floats, and decimals can be converted to elixir decimals if configured accordingly

Reply:
Then this won't solve our problem globally

Reply:
why not? we will always use decimals in our schema, and if we enable option use_decimal, they will always be converted to elixir decimals

Reply:
it's not working for us, because we have a difficult type with naming fields